### DISCUSSION TITLE
Emacs for vscode

### DISCUSSION THREAD

**Discussion by donjuanplatinum** [microsoft/vscode-discussions#205] [Extension Development QnA] [answered]
As a loyal user of emacs and vscode, I found that the current emacs plug-in does not fully adapt to emacs and vscode. I hope the vscode team can create a fully compatible editor for emacs

**Comment by alefragnani** [comment, marked as answer]
Hi @donjuanplatinum ,

I guess the comment you received in the issue you opened in VS Code repo (https://github.com/microsoft/vscode/issues/164872#issuecomment-1295503816) was in fact, to open an issue in the extension's repo, instead of here.

I'm not sure which Emacs extension are you using, but if there is something the extension itself needs VS Code to provide (a new/updated API or something like that), the extension's author could contact the VS Code team about it.

Similar requests exists for improvements in the Vim support, and I remember that from time to time some improvements in VS Code to better support it.

Hope this helps.
